### ISSUE TITLE
[0.9.6.1] CLI command line argument "precision" does not work 

### ISSUE THREAD

**Issue by avneradania** [influxdata/influxdb#5300]
Hi,

When running the following(from Linux Shell):
```
[root@node1 ~]# influx -precision rfc3339
```
I expect to see timestamp format:
```
2016-01-07T12:26:28Z	node1	96.93877410888672
2016-01-07T12:26:38Z	node1	97.93814086914062
```
Instead, i'm getting the following:
```
1452169187000000000	node1	97.93814086914062
1452169197000000000	node1	97.93814086914062
```
When running the following form in Influx Shell it works fine:
```
[root@node1 ~]# influx
Visit https://enterprise.influxdata.com to register for updates, InfluxDB server management, and monitoring.
Connected to http://localhost:8086 version 0.9.6.1
InfluxDB shell 0.9.6.1
> use my_db
>  precision rfc3339
> select * from "cpu.usage"
2016-01-07T15:22:58Z	node1	97.93814086914062
2016-01-07T15:23:00Z	node1	97.93814086914062
```

Thanks,
Avner

**Comment by otoolep** [comment]
This will be fixed by https://github.com/influxdata/influxdb/pull/5129

**Comment by e-dard** [comment]
Think this can be closed now @otoolep

**Comment by jwilder** [comment]
Fixed via #5129
